### PR TITLE
Add a reference "Code Review Developer Guide" from Google's Engineering Practices documentation.

### PR DESCRIPTION
**Pull request by gatorsmile**
This PR is to add a reference from Google's Engineering Practices documentation to our Code Review Criteria section. 

See [Code Review Developer Guide](https://google.github.io/eng-practices/review/)

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @cloud-fan @rxin @mateiz @srowen

**Review by srowen — APPROVED** [review]

File: contributing.md [hunk 1]
```diff
@@ -189,8 +189,11 @@ that maintainability, consistency, and code quality come first. New algorithms s
 <h3>Code Review Criteria</h3>
 
 Before considering how to contribute code, it's useful to understand how code is reviewed, 
-and why changes may be rejected. Simply put, changes that have many or large positives, and 
-few negative effects or risks, are much more likely to be merged, and merged quickly. 
+and why changes may be rejected. See the 
+<a href="https://google.github.io/eng-practices/review/">detailed guide for code reviewers</a> 
+from Google's Engineering Practices documentation. 
+Simply put, changes that have many or large 
+positives, and few negative effects or risks, are much more likely to be merged, and merged quickly.
```
Comment (dongjoon-hyun) [inline contributing.md:196]: Could you generate the corresponding HTML file?

**Review by holdenk — COMMENTED** [review]
I'm glad were getting more discussion around improving our code reviews. I am curious as to the Google guide being picked since the language is fairly different than we use in our project, but I'm +1 on us starting with this and iterating.

I'm also +1 on @dongjoon-hyun's suggestion to generate the HTML.

**Comment by dongjoon-hyun** [comment]
Gentle ping, @gatorsmile . :)

**Comment by gatorsmile** [comment]
https://google.github.io/eng-practices/review/reviewer/ is a very general guide. It is not for a specific language.

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

**Pull request closed without merging**